modules/apps/transfer/types/transfer_authorization.go

```diff
@@ -32,7 +35,9 @@ func (a TransferAuthorization) MsgTypeURL() string {
 }
 
 // Accept implements Authorization.Accept.
-func (a TransferAuthorization) Accept(ctx sdk.Context, msg sdk.Msg) (authz.AcceptResponse, error) {
+func (a TransferAuthorization) Accept(ctx context.Context, msg proto.Message) (authz.AcceptResponse, error) {
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
```

Comment:
can you move this to where its used?

Reply:
.... maybe?  will investigate

Reply:
this is a nit, so feel free to ignore. In the sdk we try to follow that where things are used they get unwrapped or checked, that way its easier to read